tests/Range.test.js

```diff
@@ -93,11 +93,12 @@ describe('Range', () => {
     }
     const wrapper = mount(<TestParent/>);
 
-    expect(wrapper.instance().getSlider().state.bounds.length).toBe(3);
-    expect(wrapper.find('.rc-slider-handle > .rc-slider-handle').length).toBe(3);
+    expect(wrapper.instance().getSlider().state.bounds).toHaveLength(3);
+    expect(wrapper.find('.rc-slider-handle > .rc-slider-handle')).toHaveLength(3);
     wrapper.setState({ value: [2, 4] });
-    expect(wrapper.instance().getSlider().state.bounds.length).toBe(2);
-    expect(wrapper.find('.rc-slider-handle > .rc-slider-handle').length).toBe(2);
+    wrapper.update();
```

Comment:
I'm updating the wrapper so that the last expectation L101 will work. Otherwise it is kept to 3. Are those tests really useful (considering that testing the state would be enough) ?